snap/integrity/dmverity/veritysetup.go

```diff
@@ -0,0 +1,76 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2023 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package dmverity
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/osutil"
+)
+
+// Info represents the dm-verity related data that:
+// 1. are not included in the superblock which is generated by default when running
+//    veritysetup.
+// 2. need their authenticity verified prior to loading the integrity data into the
+//    kernel
+//
+// For now, since we are keeping the superblock as it is, this only includes the root hash.
```

Comment:
please look at go doc output and golang documentation to make this format nicely. Or maybe it's go version issue?

Reply:
Not sure what to fix here. I used `go doc -all` inside the dmverity folder to test this. That's the output I get:
```
TYPES

type Info struct {
	RootHash string `json:"root-hash"`
}
    Info represents the dm-verity related data that:
     1. are not included in the superblock which is generated by default when
        running veritysetup.
     2. need their authenticity verified prior to loading the integrity data
        into the kernel.

    For now, since we are keeping the superblock as it is, this only includes
    the root hash.
```

The newline doesn't seem to be ignored in this case.